src/ast/postprocessing/structured_documentation_reconstruction.ts

```diff
@@ -129,7 +129,7 @@ export class StructuredDocumentationReconstructingPostprocessor implements ASTNo
 
     process(node: ASTNode, context: ASTContext, sources?: Map<string, string>): void {
         if (!this.isSupportedNode(node)) {
-            throw new Error(`Supplied node "${node.constructor.name}" is not supported`);
+            return;
```

Comment:
Why did the behavior change here?

Reply:
Because not it may return `false` on regular basis. We do not yet reconstruct docs for local variables, however they are still `VariableDeclaration` instances. Postptocessor will supply any `VariableDeclaration` to the reconstructor, but we do not yet able to process local vars. Also we do not want such situation lead to crash.